Assets/Scripts/Battle/BattleGUIBuilder.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class BattleGUIBuilder : MonoBehaviour
```

Comment:
Marcin, używałeś może w projekcie Data Binding ?
Fajnie byłoby aktualizować stan na UI poprzez zmianę właściwości podstaci :).

Reply:
https://www.youtube.com/watch?v=ALNT2RZr0SQ o tutaj jest dobry przykład , moglibyśmy w ten sposób zrobić :)...

Reply:
Nie wiem dokladnie o co Ci chodzi z tym Data Binding :P. Praktycznie wszystko jest tak jak na tym tutorialu , oprocz dania ustawiania w Update( trzeba zrobic tak by menu reagowalo na zmiane a nie naprzac w Updejcie). Trzeba jeszcze ustawic by sie zmienial wizard w zaleznosci od tego czyja jest tura(albo mozna dac dwa gui i je przelaczac w zaleznosci od tego czyja tura. Trzeba tez bedzie wydzielic poczatkowa inicjalizacje( te ktore rzeczy beda nie zmienialne plus to co trzeba bedzie zmieniac).

Reply:
Chodziło mi o to, że podczas zadawania obrażeń chcę zmniejszać wartość HP dla postaci. Chcę żeby kontrolka odpowiedzialna za wyświetlenie wartości HP postaci reagowała na zmianę wartości HP w tej klasie która posiada parametry :) - żeby nie trzeba było dodatkowo wywoływać metod na GUI do zmiany tekstu :). Ale jeśli jest tak jak na tutorialu to w porzadku :).